packages/app/src/HDXMultiSeriesTimeChart.tsx

```diff
@@ -580,19 +605,21 @@ export const MemoChart = memo(function MemoChart({
           tick={{ fontSize: 11, fontFamily: 'IBM Plex Mono, monospace' }}
         />
         {lines}
-        <Tooltip
-          content={
-            <HDXLineChartTooltip
-              numberFormat={numberFormat}
-              lineDataMap={lineDataMap}
-              previousPeriodOffset={previousPeriodOffset}
-            />
-          }
-          wrapperStyle={{
-            zIndex: 1,
-          }}
-          allowEscapeViewBox={{ y: true }}
-        />
+        {isClickActive == null && (
+          <Tooltip
+            content={
+              <HDXLineChartTooltip
+                numberFormat={numberFormat}
+                lineDataMap={lineDataMap}
+                previousPeriodOffset={previousPeriodOffset}
+              />
+            }
+            wrapperStyle={{
+              zIndex: 400,
```

Comment:
I think this is causing multiple tooltips to show up above flyout panels. Ex. if you go to services > click on na endpoint > hover over the request throughput chart for an endpoint

<img width="1458" height="1070" alt="image" src="https://github.com/user-attachments/assets/babf7707-b29c-4e1b-8f22-243cdf15af37" />

Reply:
As a side note, for some reason the `request throughput` `view events` tooltip does not show up when clicking on the chart in the drawer, not sure if related?

Reply:
Good catch, I changed the implementation of the tooltips to account for this modal.

The issue of not seeing the event details was these charts didn't pass a sourceId, I have added that so thats fixed too!